po/extra/it.po

```diff
@@ -0,0 +1,62 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Saverio <saverio.morelli@outlook.it>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-06-01 12:24+0200\n"
+"PO-Revision-Date: 2021-06-27 23:35+0200\n"
+"Language-Team: Italian <saverio.morelli@outlook.it>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Gtranslator 40.0\n"
+"Last-Translator: Saverio <saverio.morelli@outlook.it>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"Language: it\n"
+
+#: data/com.github.rajkumaar23.badgie.desktop.in:3
+#: data/com.github.rajkumaar23.badgie.appdata.xml.in:7
+msgid "Badgie"
+msgstr "Badgie"
+
+#: data/com.github.rajkumaar23.badgie.desktop.in:4
+msgid "Badge Generator"
+msgstr "Generatore di distintivi"
+
+#: data/com.github.rajkumaar23.badgie.desktop.in:5
+msgid ""
+"Generate realtime badges from PlayStore data for your android app README"
+msgstr ""
+"Genera distintivi in tempo reale dai dati del PlayStore per il README della "
+"tua app Android"
+
+#: data/com.github.rajkumaar23.badgie.desktop.in:8
+msgid "com.github.rajkumaar23.badgie"
+msgstr "com.github.rajkumaar23.badgie"
+
+#: data/com.github.rajkumaar23.badgie.desktop.in:11
+msgid "Playstore;Badge;"
+msgstr "Playstore;Distintivo;"
+
+#: data/com.github.rajkumaar23.badgie.appdata.xml.in:8
+msgid "Badges for your android app's README"
+msgstr "Distintivo per il README della tua app Android"
+
+#: data/com.github.rajkumaar23.badgie.appdata.xml.in:10
+msgid ""
+"Say NO to hardcoding! Generate badges for your android app's README with "
+"realtime data from Google Playstore such as Version, Number of Installs, "
+"Rating etc"
+msgstr ""
+"Dici NO all'hardcoding! Genera distintivi per il README della tua app "
```

Comment:
```suggestion
"Dici NO alla hardcoding! Genera distintivi per il README della tua app "
```
This is what Google Translate suggests me. Please let me know if the suggestion is wrong.

Reply:
The suggestion by Google is not wrong, but it's the "long form". We in Italian use, basically, just the first one.

It's like "I am" and "I'm" > Google will suggest always "I am", but "I'm" is correct as well

Reply:
Gotcha! Merging it 🎉